### PR TITLE
🌱 Bump CAPI to v1.6.1

### PR DESCRIPTION
**Pull request by Sunnatillo**
as title

### CODE REVIEW

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
Cherry-pick to 1.6 branch?

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.6.0 --bootstrap kubeadm:v1.6.0 \
+   clusterctl init --core cluster-api:v1.6.1 --bootstrap kubeadm:v1.6.0 \
```
Comment (kashifest) [inline README.md:50]:
```suggestion
   clusterctl init --core cluster-api:v1.6.1 --bootstrap kubeadm:v1.6.1 \
   --control-plane kubeadm:v1.6.1 -v5
```

**Review by kashifest — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sunnatillo) [reply README.md]: done

**Review by kashifest — APPROVED** [review]

**Review by tuminoid — COMMENTED** [review]
/lgtm

/test-centos-e2e-integration-main
/test-ubuntu-integration-main

**Pull request merged**